ios/brave-ios/Sources/Brave/Frontend/Browser/New Tab Page/NewTabPageViewController.swift

```diff
@@ -535,12 +538,14 @@ class NewTabPageViewController: UIViewController {
     }
     videoPlayer?.didFinishPlaybackEvent = { [weak self] in
       guard let self = self else { return }
-      self.videoButtonsView.isHidden = true
+      self.fadeInCollectionViewAndShowVideoButtons()
       self.reportSponsoredBackgroundEvent(.media100)
-      self.showAndFadeInCollectionView()
+      self.videoPlayer?.seekToStopFrame()
```

Comment:
nit: move to `reportSponsoredBackgroundEvent` after `self.videoPlayer?.seekToStopFrame()`

Reply:
Fixed